From bf17558c7afb78c5d86419738ff93ef110663f97 Mon Sep 17 00:00:00 2001
From: Mathieu Desnoyers <mathieu.desnoyers@polymtl.ca>
Date: Tue, 12 Aug 2008 10:32:56 -0400
Subject: [PATCH] lttng-relay-hotplug

LTTng relay hotplug

Change relay priority so ltt heartbeat can be run _after_ relay buffer setup.

Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@polymtl.ca>
CC: Tom Zanussi <tzanussi@gmail.com>
---
 kernel/relay.c |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/kernel/relay.c b/kernel/relay.c
index 8d13a78..60cf081 100644
--- a/kernel/relay.c
+++ b/kernel/relay.c
@@ -1354,8 +1354,7 @@ EXPORT_SYMBOL_GPL(relay_file_operations);
 
 static __init int relay_init(void)
 {
-
-	hotcpu_notifier(relay_hotcpu_callback, 0);
+	hotcpu_notifier(relay_hotcpu_callback, 5);
 	return 0;
 }
 
-- 
1.5.5.1

